codec-http2/src/main/java/io/netty/handler/codec/http2/Http2Exception.java

```diff
@@ -229,6 +251,19 @@ public int streamId() {
         }
     }
 
+    public static final class HeaderListSizeException extends StreamException {
+        private final boolean decode;
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Make this "decode" field final. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://garage.netty.io/sonarqube/coding_rules#rule_key=squid%3AS1165)